src/scripts/components/time-slider/time-slider.tsx

```diff
@@ -89,39 +92,68 @@ const TimeSlider: FunctionComponent = () => {
     };
   };
 
+  const setTimeOutput = (targetValue: string, min: number, max: number) => {
```

Comment:
If you only want to set the style on an element you can do it with a normal inline style attribute and don't need to use `useRef`